lego/apps/users/tasks.py

```diff
@@ -0,0 +1,80 @@
+from datetime import timedelta
```

Comment:
You could save an import by using this from `django.utils.timezone` (Or just use `timezone.timedelta`)